chasm/lib/scheduler/invoker_tasks.go

```diff
@@ -140,6 +142,19 @@ func (e *InvokerExecuteTaskExecutor) Execute(
 				compiledSpec:       s.compiledSpec,
 			}
 
+			lcs, err := s.LastCompletionResult.Get(ctx)
+			if err != nil {
+				return struct{}{}, err
```

Comment:
I suggested creating a `type NoValue = *struct{}` in the `chasm` package, like we did in `nexus`.
I feel like it's nicer to use `func() (chasm.NoValue, error) { return nil, err }` than the empty struct approach.
Alternatively we can add a `ReadComponentRetErr` helper. But feel like the former option is nicer.
Anyways, definitely not blocking the PR, just something to consider for the future.